### PR TITLE
Feature/choose applicant stip types

### PR DESCRIPTION
**Pull request by Mihran9991**

### CODE REVIEW

File: examples/index.html [hunk 1]
```diff
@@ -95,27 +114,49 @@
 
   viq.enableLogging(true);
 
-  var renderBtn = document.querySelector("#render");
+  renderOptions(applicantTypeNode, Object.values(VerifyIQ.ApplicantTypes));
```
Comment (shahen94) [inline examples/index.html:117]: This example will not work on IE

Object.values are not supported on IE11

File: src/sdk.ts [hunk 2]
```diff
@@ -182,8 +184,16 @@ class VerifyIQ implements IVerifyIQ {
    * @param htmlElement {HTMLElement}
    * @param applicationId {String}
    */
-  public renderApplicationId(htmlElement: HTMLElement, applicationId: string) {
+  public renderApplicationId({
+    htmlElement,
+    applicationId,
+    applicant = VerifyIQ.ApplicantTypes.PrimaryApplicant,
+    stipulation = ''
+  }: renderApplicationIdArgType) {
```
Comment (shahen94) [inline src/sdk.ts:192]: Type name is confusing. `renderApplicationIdArgType `

Let's rename it to `RenderByApplicationIdParams`

**Review by shahen94 — CHANGES_REQUESTED** [review]
Overall looks good to me.

Let's made small changes

File: src/sdk.ts (hunk 2 above)
Comment (Mihran9991) [reply src/sdk.ts]: @shahen94 updated!

**Review by shahen94 — APPROVED** [review]

**Review by mkim177 — COMMENTED** [review]
@Mihran9991 overall looks good, but we do need to make sure this works on IE11

**Pull request merged**